src/components/navigation/problem-tab-content.tsx

```diff
@@ -55,16 +55,23 @@ export const ProblemTabContent: React.FC<IProps>
 
   }, [ui, ui.focusDocument]);
 
-  const handleTabClick = (titleArgButReallyType: string, typeArgButReallyTitle: string) => {
-    // TODO: this function has its argument names reversed (see caller for details.)
-    // We can't simply switch it, however, because that would introduce a breaking change
-    // in the log event stream, so for now we just rename the arguments for clarity.
+  const handleTabSelected = (index: number) => {
+    const section = sections?.[index];
+    if (!section) { return; }
```

Comment:
```suggestion
    if (!section) return;
```

Reply:
I'm not a fan of single line `if`s without braces because of the potential for a developer to try to turn it into a multiline `if` without remembering the braces. I checked and there are a far amount (17) of single line `if`s with braces. A regex to find them is `^[^\/]*if .* \{ return`.  There are definitely many more single line `if`s without braces (236). Regex for this is `^[^\/]*if [^{]* return`.

This is similar to the issue that I think bit Bill here: https://concord-consortium.slack.com/archives/C035J6RDAK0/p1666296973289279 He added a console.log to see what was going on but then didn't add the extra syntax to return result of debounce. In that case the conciseness of doing `() => debounce` compared to `() => { return debounce...` seem to outweigh the potential risk. But in the case of an if statement it is only a couple more chars and spaces.

But I'll go with consistency here 236 is way more than 17. 😄

Reply:
>I'm not a fan of single line ifs without braces because of the potential for a developer to try to turn it into a multiline if without remembering the braces.

Personally, I think the (small) readability gain is more salient than the (small) potential for multiline confusion. In the case you mention, I think the confusion was more about the use of `debounce` than the braces.

Also just noticed that there's a warning in the same file that you could take care of while you're in the neighborhood:
```
React Hook useEffect has a missing dependency: 'findSelectedSectionIndex'.
```